runtime/util/match_log.cpp

```diff
@@ -40,12 +40,13 @@ void addMatchSuccess(void) {
        nullptr});
 }
 
-void addMatchFailReason(void *subject, char *pattern, char *sort) {
+void addMatchFailReason(void *subject, const char *pattern, const char *sort) {
```

Comment:
Looks like we use both `const char *` and `char const*` in various places.

We should consider setting set `QualifierAlignment: Right` to clean this up: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#qualifieralignment

Reply:
@gtrepta `const char *` and `char const *` are equivalent - both are a pointer to a constant `char`. I think the common lingo here is "west const" versus "east const". 

(Both are distinct from `char * const` though - a constant pointer to a `char`.)

The warning is worth bikeshedding about, but IMO it's not a big concern:
- I've used it on large production codebases before with no issues
- Looking through the git blame history that adds this option `clang-format`, AFAICT the only known issue is that lower-case macros may be mis-identified as types, meaning that it could rewrite
```
#define intptr int *
const intptr a;
```
to
```
intptr const a;
```
which is a semantic change. This doesn't seem to appear anywhere in our codebase, and if it does, we should just change it to a typedef instead of a macro anyways.

Reply:
I think that `clang-tidy` will actually catch usages of lower-case macros like that anyway, so I think we can make this change safely.